src/librbml/lib.rs

```diff
@@ -772,11 +924,29 @@ pub mod writer {
             let cur_pos = try!(self.writer.tell());
             try!(self.writer.seek(last_size_pos as i64, old_io::SeekSet));
             let size = cur_pos as uint - last_size_pos - 4;
-            try!(write_sized_vuint(self.writer, size, 4));
-            let r = try!(self.writer.seek(cur_pos as i64, old_io::SeekSet));
+
+            // relax the size encoding for small tags (bigger tags are costly to move).
```

Comment:
Is "relaxing" the process of reencoding an integer as a smaller integer? (If so, it seems like this might be better described as "tightening" or something, it's not really very 'relaxing'.)

(Total nitpick.)